standup_notes/__main__.py

```diff
@@ -124,6 +164,36 @@ def iterate_weekday(day: date, func: Callable[[date], date]) -> date:
         return iterate_weekday(next_day, func)
 
 
+def delete_notes(amount):
+    file_to_delete = []
+    if len(str(amount)) != 8:
+        print("Please enter in a valid date")
+        return
+    for file in os.listdir(STANDUP_NOTES):
+        value = int(file.split('.')[0])
+        if value < amount:
+            file_to_delete.append(file)
+    if file_to_delete:
```

Comment:
nitpick: can you name this "files_to_delete" since it's a list?

Reply:
Sure